File: YSDB/MSDB/Models/flowInstance.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace MSDB.Models
{
    public class flowInstance : tablebase
    {
        [Key]
        public int id { get; set; }
        public int? curNodeId { get; set; }
        public int flowId { get; set; }

        ///<summary>
        /// 0：进行中 1:通过 2：驳回 3:激活
        ///</summary>
        public int status { get; set; }

        ///<summary>
        /// 0进行中 1通过 2驳回 3激活 4提交 5未提交 6失败 7提交失败
        ///</summary>
        public int? nodeStatus { get; set; }

        public int moduleId { get; set; }
        public int deptId { get; set; }

        public int lastSubmiter { get; set; }

        public int? companyId { get; set; }

        ///<summary>
        ///内容明细  
        ///</summary>
        [MaxLength(200)]
        public string content { get; set; }

        ///<summary>
        /// 审核层级（连续多级主管审核-当前审核级数）
        public int? auditLevel { get; set; }

        ///<summary>
        ///当前审核用户
        ///</summary>
        [MaxLength(200)]
        public string auditUsers { get; set; }
        ///<summary>
        ///截至日期，null为无期限
        ///</summary>
        public DateTime? expired { get; set; }

        ///<summary>
        /// 可撤回 (提交后默认为true，审核后置为false)
        ///<summary>
        public bool revokable{ get; set; }
    }
}

File: MS/src/main/java/com/yskc/ms/entity/ms/FlowInstance.java
package com.yskc.ms.entity.ms;

import com.baomidou.mybatisplus.annotations.TableId;
import com.baomidou.mybatisplus.annotations.TableName;

import java.util.Date;

/**
 * Created by hck
 * on 2020/12/15 10:35
 * description:流程实例
 */
@TableName("flowInstance")
public class FlowInstance {
    @TableId
    private Integer id;
    private Integer creatorId;
    private Integer lastUpdatorId;
    private Date createTime;
    private Date lastUpdateTime;
    @Deprecated
    private Integer curNodeId;
    private Integer flowId;
    private Integer status;//实例状态：0进行中1通过2驳回3激活
    private Integer moduleId;
    @Deprecated
    private Integer nodeStatus;//节点状态 0进行中1通过2驳回3激活
    private Integer deptId;
    private Integer lastSubmiter;
    private Integer companyId;
    private String content;
    private Date expired;
    private Integer auditLevel;
    private Boolean revokable;
    public FlowInstance() {
    }

    public Integer getModuleId() {
        return moduleId;
    }

    public void setModuleId(Integer moduleId) {
        this.moduleId = moduleId;
    }

    public Integer getNodeStatus() {
        return nodeStatus;
    }

    public void setNodeStatus(Integer nodeStatus) {
        this.nodeStatus = nodeStatus;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public Integer getCreatorId() {
        return creatorId;
    }

    public void setCreatorId(Integer creatorId) {
        this.creatorId = creatorId;
    }

    public Integer getLastUpdatorId() {
        return lastUpdatorId;
    }

    public void setLastUpdatorId(Integer lastUpdatorId) {
        this.lastUpdatorId = lastUpdatorId;
    }

    public Date getCreateTime() {
        return createTime;
    }

    public void setCreateTime(Date createTime) {
        this.createTime = createTime;
    }

    public Date getLastUpdateTime() {
        return lastUpdateTime;
    }

    public void setLastUpdateTime(Date lastUpdateTime) {
        this.lastUpdateTime = lastUpdateTime;
    }

    public Integer getCurNodeId() {
        return curNodeId;
    }

    public void setCurNodeId(Integer curNodeId) {
        this.curNodeId = curNodeId;
    }

    public Integer getFlowId() {
        return flowId;
    }

    public void setFlowId(Integer flowId) {
        this.flowId = flowId;
    }

    public Integer getStatus() {
        return status;
    }

    public void setStatus(Integer status) {
        this.status = status;
    }

    public Integer getDeptId() {
        return deptId;
    }

    public Integer getLastSubmiter() {
        return lastSubmiter;
    }

    public void setLastSubmiter(Integer lastSubmiter) {
        this.lastSubmiter = lastSubmiter;
    }

    public void setDeptId(Integer deptId) {
        this.deptId = deptId;
    }

    public Integer getCompanyId() {
        return companyId;
    }

    public void setCompanyId(Integer companyId) {
        this.companyId = companyId;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }

    public Boolean getRevokable() {
        return revokable;
    }

    public void setRevokable(Boolean revokable) {
        this.revokable = revokable;
    }

    public Date getExpired() {
        return expired;
    }

    public void setExpired(Date expired) {
        this.expired = expired;
    }

    public Integer getAuditLevel() {
        return auditLevel;
    }

    public void setAuditLevel(Integer auditLevel) {
        this.auditLevel = auditLevel;
    }
}
